mobile/examples/object_detection/ios/README.md

```diff
@@ -0,0 +1,49 @@
+# ONNX Runtime Mobile object detection iOS sample application
+
+This is an example app using object detection which is able to continuously detect the objects in the frames seen by your iOS device's back camera and display the detected object bounding boxes, detected class and corresponding inference confidence on the screen.
+
+This example is heavily based on [Google Tensorflow lite - Object Detection Examples](https://github.com/tensorflow/examples/)
+
+### Model
+We use pre-trained quantized MobileNet SSD V1 model in this sample app. 
+
+The original `ssd_mobilenet_v1.tflite` model can be downloaded [here](https://www.tensorflow.org/lite/examples/object_detection/overview#get_started)
+
+## Requirements
+- Install Xcode 12.5 and above (preferably latest version)
+- A valid Apple Developer ID
+- A real iOS device with a camera (preferably iphone 12/iphone 12 pro)
+- Xcode command line tools `xcode-select --install`
+- Clone the `onnxruntime-inference-examples` source code repo
+
+## Build And Run
+
+1. Install CocoaPods. `sudo gem install cocoapods`
+
+2. Run `pod install` to generate the workspace file under `<ONNXRuntime-inference-example-root>/mobile/examples/object_detections/ios/`. 
+- At the end of this step, you should get a file called `ORTObjectDetection.xcworkspace`.
+
+3. Run download script `download.sh` under `<ONNXRuntime-inference-example-root>/mobile/examples/object_detections/ios/ORTObjectDetection/`. The script will download an original tf model along with the model metadata `labelmap.txt` and convert the model to onnx and then further convert it to ort format model (the format can be executed on mobile applications).
```

Comment:
nit: as a naive user I need a bit more info here, like where do I run each command from. is it from a terminal window or within xcode? if a terminal, do commands like `pod install` need to run from a specific directory?